src/settings/tabrepository.h

```diff
@@ -0,0 +1,20 @@
+#ifndef TABREPOSITORY_H
+#define TABREPOSITORY_H
+
+#include <QWidget>
+
+class TabRepository : public QWidget
+{
+    Q_OBJECT
+public:
+    explicit TabRepository(QWidget *parent = nullptr);
+private:
+    // design parameters
+    int mBasicStretch                = 1;
```

Comment:
why is the space between variable and value so large?

Reply:
check it in tabconnection.h